cinema/views.py

```diff
@@ -56,3 +94,62 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
+
+        date = self.request.query_params.get("date")
+        movie = self.request.query_params.get("movie")
+
+        if date:
+            [year, month, day] = date.split("-")
+            queryset = queryset.filter(
+                show_time__year=year,
+                show_time__month=month,
+                show_time__day=day
+            )
```

Comment:
You can use `show_time__date=date` instead. Just use built-in datetime to transform date